packages/go_router_builder/lib/src/route_config.dart

```diff
@@ -88,7 +88,8 @@ class RouteConfig {
     }
 
     // TODO(kevmoo): validate that this MUST be a subtype of `GoRouteData`
-    final ClassElement classElement = typeParamType.element;
+    //
```

Comment:
I assume deleting the TODO was a mistake?

Reply:
Oops, it was a fat fingered return key hit. Fixed.